intelmq/bots/collectors/github_api/_collector_github_api.py

```diff
@@ -23,16 +22,15 @@
 
 
 class GithubAPICollectorBot(CollectorBot):
-    basic_auth_username = None
-    basic_auth_password = None
+    personal_access_token: str = None
 
     def init(self):
         if requests is None:
-            raise ValueError('Could not import requests. Please install it.')
+            raise MissingDependencyError("requests")
 
         self.__user_headers = static_params['headers']
-        if self.basic_auth_username is not None and self.basic_auth_password is not None:
-            self.__user_headers.update(self.__produce_auth_header(self.basic_auth_username, self.basic_auth_password))
+        if self.personal_access_token is not None:
```

Comment:
Treating empty string equally to None
```suggestion
        if self.personal_access_token:
```